Makefile

```diff
@@ -0,0 +1,115 @@
+# --- Global -------------------------------------------------------------------
+O = out
+COVERAGE = 80
+SEMVER ?= $(shell git describe --tags --dirty --always)
+COMMIT_SHA ?= $(shell git rev-parse --short HEAD)
+
+all: build test check-coverage lint  ## build, test, check coverage and lint
+	@if [ -e .git/rebase-merge ]; then git --no-pager log -1 --pretty='%h %s'; fi
+	@echo '$(COLOUR_GREEN)Success$(COLOUR_NORMAL)'
+
+clean::  ## Remove generated files
+	-rm -rf $(O)
+
+.PHONY: all clean
+
+# --- Build --------------------------------------------------------------------
+GO_LDFLAGS = \
+	-X main.Semver=$(SEMVER) \
+	-X main.CommitSha=$(COMMIT_SHA)
+
+build: | $(O)  ## Build jcdc binary to out/
+	go build -o $(O)/jcdc -ldflags='$(GO_LDFLAGS)' .
+
+install:  ## Build and install jcdc
+	go install -ldflags='$(GO_LDFLAGS)' .
+
+run: build  ## Run jcdc server
+	$(O)/jcdc
+
+.PHONY: build install run
+
+# --- Test ---------------------------------------------------------------------
+COVERFILE = $(O)/coverage.txt
+
+test: | $(O) ## Run tests and generate a coverage file
```

Comment:
nit: we've been using a double-space before `##`

Reply:
done.